### PR TITLE
Clean up of some obvious syntax items

### PR DESCRIPTION
**Pull request by SCoureFoure**
Just reading through and made the changes as I went. It should not change functionality at all.

### CODE REVIEW

File: appium/lib/screen-object/accessors/element.rb [hunk 1]
```diff
@@ -239,23 +231,29 @@ def get_element_by_exact_text(value)
         end
       end
 
+      def is_a_string?(argument)
```
Comment (mike-herbert) [inline appium/lib/screen-object/accessors/element.rb]: This is unnecessary, remove and refactor uses of this method to be `argument.is_a? String`

File: appium/lib/screen-object/accessors/element.rb [hunk 2]
```diff
@@ -72,32 +71,30 @@ def element_attributes
       def get_position
         my_element = element.rect
```
Comment (mike-herbert) [inline appium/lib/screen-object/accessors/element.rb]: `my_element` is a weird name to choose 🤷‍♂

File: appium/lib/screen-object/accessors/element.rb [hunk 3]
```diff
@@ -72,32 +71,30 @@ def element_attributes
...
       end
 
       def dynamic_xpath(text)
-        concat_attribute=[]
+        concat_attribute = []
         element_attributes.each{|i| concat_attribute << %Q(contains(@#{i}, '#{text}'))}
-        puts  "//#{locator[0]}[#{concat_attribute.join(' or ')}]"
-        locator1="xpath~//#{locator[0]}[#{concat_attribute.join(' or ')}]"
-        @locator=locator1.split("~")
+        puts "//#{locator[0]}[#{concat_attribute.join(' or ')}]"
```
Comment (mike-herbert) [inline appium/lib/screen-object/accessors/element.rb]: Remove `puts` statement

File: appium/lib/screen-object/accessors/element.rb (hunk 2 above)
Comment (SCoureFoure) [reply appium/lib/screen-object/accessors/element.rb]: changed

File: appium/lib/screen-object/accessors/element.rb (hunk 3 above)
Comment (SCoureFoure) [reply appium/lib/screen-object/accessors/element.rb]: removed

**Comment by SCoureFoure** [comment]
A lot got changed on the accessors.rb, but none of it was really to the methods that existed outside the comments, and I added methods that were missing.

**Review by sandeepnagra — APPROVED** [review]
Looks good to me

**Pull request merged**